src/main/java/com/cloudogu/smeagol/wiki/domain/WikiRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.cloudogu.smeagol.wiki.domain;
+
+import java.util.Optional;
+
+/**
+ * Wiki repositories are able to query wiki objects.
+ */
+public interface WikiRepository {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Annotate the "WikiRepository" interface with the @FunctionalInterface annotation [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS1609)